README.md

```diff
@@ -0,0 +1,166 @@
+
+#### Writers vs. Reducers
+In `redux` you have a `reducer` that takes the current state and an action which it processes to return a new state. In `realm-react-redux` there is a similar concept: the `writer`. A `writer` takes a `Realm` instance and an action and writes to the `Realm` db. The `store` handles wrapping the entire thing in a single `realm.write()` transaction so your writers can focus on handling only creates/updates/deletes it needs. There is also a `combineWriters()` method which takes an array of writers and combines them into a single one for the store, very similar to `combineReducers()` in `redux`.
+
+#### Provider
+`react-redux` was designed with extensibility in mind so we can actually use its `Provider` class as is. As a convenience there is a `RealmProvider` exposed to handle setting up the proper `storeKey`, this is probably what you want to use.
+
+#### Connect
+Again, `react-redux` is quite extensible so we can re-use a lot of the libraries core code. However, in order to work with `Realm` the `connect()` API needed to be changed. 
+
+| `connect`     | `realmConnect` |
+| ------------- | -------------  |
+| --  | `mapPropsToQueries(realm, ownProps)`   |
+| `mapStateToProps(state, ownProps)`  | `mapQueriesToProps(queries, ownProps)` |
+| `mapDispatchToProps(dispatch, ownProps)`  | `mapRealmDispatchToProps(dispatch, ownProps)` |
+| `mergeProps(stateProps, dispatchProps, ownProps)`  | `mergeProps(queryProps, dispatchProps, ownProps)` |
+
+## API Docs
+#### Coming soon!
+
+## Getting Started
+#### Installation
+ A real npm package is coming soon, but to start using right away:
+`npm install https://github.com/lolatravel/realm-react-redux/tarball/v0.0.5`
+
+#### Running the examples
+There's only a basic example right now, more coming soon.
+
+```sh
+git clone git@github.com:lolatravel/realm-react-redux.git
+cd realm-react-redux
+npm install
+cd examples/basic_todo_example
+npm install
+# if you don't have react-native-cli installed globally
+npm install -g react-native-cli
+react-native run-ios
+```
+
+### Basic Example
+```javascript
+// models.js
+export class ToDo {
+    static schema = {
+        name: 'ToDo',
+        primaryKey: 'id',
+        properties: {
+            id: { type: 'string' },
+            name: { type: 'string' },
+            completed: { type: 'bool', default: false }
+        }
+    };
+}
+```
+```javascript
+// actions.js
+export const CREATE_TODO = 'CREATE_TODO';
+export const TOGGLE_TODO = 'TOGGLE_TODO';
+
+export function createTodo(name) {
+    return {
+        type: CREATE_TODO,
+        name
+    };
+}
+
+export function toggleTodo(id) {
+    return {
+        type: TOGGLE_TODO,
+        id
+    };
+}
+```
+```javascript
+// writers.js
+import uuid from 'uuid/v4';
+import { CREATE_TODO, TOGGLE_TODO } from './actions';
+
+export function todoWriter(realm, action) {
```

Comment:
It might be worth calling out a difference between writers and reducers: Writers are side-effectful and do not return a new value, while reducers are pure functions. This might be an academic distinction, so I'll leave it up to you.